sprout/celery_runner

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python3
 from __future__ import absolute_import, unicode_literals
```

Comment:
`__future__` imports still necessary? Or is this some django thing?

Reply:
I missed that. my bad